drivers/potentiometer/ad514x/ad514x_user_config.c

```diff
@@ -0,0 +1,127 @@
+/***************************************************************************//**
+ * @file   ad514x_user_config.c
+ * @brief  User configurations for AD514x No-OS drivers
+********************************************************************************
+* Copyright (c) 2024 Analog Devices, Inc.
+* All rights reserved.
+*
+* This software is proprietary to Analog Devices, Inc. and its licensors.
+* By using this software you agree to the terms of the associated
+* Analog Devices Software License Agreement.
+*******************************************************************************/
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include "dpot_user_config.h"
+
+/******************************************************************************/
+/********************* Macros and Constants Definition ************************/
+/******************************************************************************/
+
+/******************************************************************************/
+/******************** Variables and User Defined Data Types *******************/
+/******************************************************************************/
+
+/* AD5141 No-OS driver init parameters */
+struct ad5141_dpot_init_param ad5141_init_params = {
+	/* Select interface type */
+	.eintf_type = AD_SPI_INTERFACE,
+	/* Define interface init parameters */
+	.spi_init = &spi_init_params,
+	.i2c_init = &i2c_init_params,
```

Comment:
`i2c_init_params` is application specific, this shouldn't be in driver code, i think you're trying to get it from `"dpot_user_config.h"` but that's not and okay way to share info with the device driver, all info should go through init_param when initializing the dpot